spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -719,6 +714,12 @@ def run(self, run_time):
                 self._do_load()
                 loading_done = True
 
+        # Disable auto pause and resume if the binary can't do it
+        if (self._executable_start_type !=
+                ExecutableStartType.USES_SIMULATION_INTERFACE):
+            self._config.set("Buffers", "use_auto_pause_and_resume",
+                             "False")
```

Comment:
2. executable start type has no position in a virtual board run, thus not being set is the right thing to do. your not going through the loading or running phase, so no point setting a param for no reason. separation of concerns and all that. To set it, you need to find the binaries, which is what were trying to avoid when using a virtual board. 

1. ill need to look at in my ide, as trying to traverse the gituhb equiv is not helping. will look at after the dma review